File: fabric/src/main/java/net/impleri/playerskills/server/fabric/PlayerSkillsServerFabric.java
package net.impleri.playerskills.server.fabric;

import net.fabricmc.api.DedicatedServerModInitializer;
import net.impleri.playerskills.server.PlayerSkillsServer;

public class PlayerSkillsServerFabric implements DedicatedServerModInitializer {
    @Override
    public void onInitializeServer() {
        PlayerSkillsServer.init();
    }
}

File: fabric/src/main/scala/net/impleri/playerskills/server/fabric/PlayerSkillsServerFabric.scala
package net.impleri.playerskills.server.fabric

import net.fabricmc.api.DedicatedServerModInitializer
import net.impleri.playerskills.server.PlayerSkillsServer

case class PlayerSkillsServerFabric() extends DedicatedServerModInitializer {
  override def onInitializeServer(): Unit = {
    PlayerSkillsServer.init()
  }
}
